rules/windows/pipe_created/pipe_created_susp_uiautomationcore_pipe_pattern.yml

```diff
@@ -0,0 +1,44 @@
+title: Suspicious UI Automation Named Pipe Creation
+id: 5e631c7d-c93e-48f5-a78a-1706d9cda972
+status: experimental
+description: |
+    Detects creation of named pipes with UIA_PIPE_ prefix, which are used by Microsoft's UI Automation framework.
+    This activity may indicate malicious use of UI Automation for credential theft or unauthorized UI interaction.
+    Coyote malware has been observed using UI Automation to extract credentials from banking applications.
+references:
+    - https://www.akamai.com/blog/security-research/active-exploitation-coyote-malware-first-ui-automation-abuse-in-the-wild#detect
+    - https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-intro
+author: Saiprashanth Pulisetti, @prashanthblogs
+date: 2025-07-29
+tags:
+    - attack.credential-access
+    - attack.t1555
+    - attack.t1056.002
+    - attack.collection
+    - attack.t1113
+logsource:
+    category: pipe_created
+    product: windows
+    definition: 'Note that you have to configure logging for Named Pipe Events in Sysmon config (Event ID 17 and Event ID 18). The basic configuration is in popular sysmon configuration (https://github.com/SwiftOnSecurity/sysmon-config), but it is worth verifying. You can also use other repo, e.g. https://github.com/Neo23x0/sysmon-config, https://github.com/olafhartong/sysmon-modular. How to test detection? You can check powershell script from this site https://svch0st.medium.com/guide-to-named-pipes-and-hunting-for-cobalt-strike-pipes-dc46b2c5f575'
+detection:
+    selection:
+        PipeName|startswith: '\UIA_PIPE_'
+    filter_main_legitimate_processes:
+        Image|startswith:
+            - 'C:\Windows\System32\'
+            - 'C:\Windows\SysWOW64\'
+        Image|endswith:
+            - '\narrator.exe'
+            - '\osk.exe'
+            - '\magnify.exe'
+            - '\utilman.exe'
+            - '\sethc.exe'
+            - '\AtBroker.exe'
+            - '\displayswitch.exe'
+            - '\dwm.exe'
```

Comment:
Seeing this amount of filters and reading the blog a bit. I can guarantee that there should stuff from program files too loading this. Hence the rule cannot be a detection but rather a hunting rule